time/unit.go

```diff
@@ -56,6 +56,14 @@ func (tu Unit) Value() (time.Duration, error) {
 	return 0, errUnrecognizedTimeUnit
 }
 
+// Count returns the number of units contained within the duration.
+func (tu Unit) Count(d time.Duration) (int, error) {
```

Comment:
nit: make this `int64` instead of `int`

Reply:
Must have missed this comment, I feel it would be more idiomatic to return an `int` instead of `int64`. For the majority of use cases I imagine it will be a noop anyhow since it'll be run on a 64 bit platform anyway.

Reply:
Sounds good